### DISCUSSION TITLE
What's the function of the toggle in the top right of the now playing queue?

### DISCUSSION THREAD

**Discussion by jrebs** [jeffvli/sonixd#441] [Q&A] [answered]
I checked the usage guide but couldn't figure out the function of this toggle widget. Can someone clear this up for me, please?

![image](https://github.com/jeffvli/sonixd/assets/4203789/4b8537c9-d875-4f8b-a2e1-027d71aedea7)

**Comment by jeffvli** [comment, marked as answer]
It enables the "Auto scroll" mode which follows the currently playing track. You can also view this checkbox on the "Now Playing" page.

**Reply by jrebs** [reply]
Thank you!
